src/llama.cpp

```diff
@@ -8658,6 +8656,10 @@ static int llama_model_load(const std::string & fname, llama_model & model, llam
         return -1;
     }
 
+    // loading time will be recalculate after the first eval, so
+    // we take page faults deferred by mmap() into consideration
+    model.t_load_us = ggml_time_us() - model.t_start_us;
```

Comment:
should this be wrapped inside `if (!ctx->cparams.no_perf)`?

Reply:
I left it in because model loading happens quite rare (usually once upon program start). The time calls in the inference loop happen more often, so we want to have means to disable those.